### DISCUSSION TITLE
금일 회의 안내

### DISCUSSION THREAD

**Discussion by on-Sync** [code-etc/Documents#7] [Announcements]
> 회의시간 단축을 위해 다음 목록에 대한 준비를 부탁드립니다.

- Docker remote repository URI (ex. 본인닉네임/이미지네임:현재버전 )
- Image versioning 전략에 대한 본인생각 또는 의견
- GitHub Actions 를 활용한 Docker hub 사용법 학습 또는 대체방안 존재시 공유
- 현재 작업 진행내용 정리
- 협의 또는 공유사항 안내

> 본 내용 확인시 업버튼을 눌러주시기 바랍니다. 감사합니다.

@daehwan2 @tjwjdgks @yunjin-kim

**Comment by tjwjdgks** [comment]
dockerhub  auth 서버 이미지 이름 예시 입니다
tjwjdgks43/code-etc-authserver:v.x.x 

auth서버 dockerhub repo <br>
https://hub.docker.com/repository/docker/tjwjdgks43/code-etc-authserver

**Comment by yunjin-kim** [comment]
dockerhub frontend 이미지 이름 예시입니다
tion6514/code-etc-frontend:v.x.x

frontend dockerhub

https://hub.docker.com/r/tion6514/code-etc-frontend
